src/test/java/com/PlaceLab/Amila/qamp/pages/LoginPage.java

```diff
@@ -0,0 +1,97 @@
+    private final static By FORGOT_PWD_CONTINUE_BUTTON = By.cssSelector("input[type=\"submit\"].btn");
+    private final static By FORGOT_PWD_NOTIFICATION_MESSAGE = By.xpath("//p[contains(text(), 'We have sent you a link to change your password')]");
+    private final static By NEW_PWD_NOTIFICATION_MESSAGE = By.id("login");
+    private final static By INVALID_CREDENTIALS = By.xpath("//div[contains(text(), 'Invalid credentials!')]");
+    private final static By LOGIN_PAGE_INV = By.xpath("//*[@id=\"login-content\"]");
+    private final WebDriver driver;
+
+    public LoginPage(final WebDriver webDriver) {
+        this.driver = webDriver;
+    }
+
+    public void validateLoginPageContent() {
+        final String actualPageTitle = driver.getTitle();
+
+        Assert.assertEquals(actualPageTitle, EXPECTED_PAGE_TITLE, "Validate page title is correct");
+        Assert.assertTrue(driver.findElement(LOGIN_PAGE).isDisplayed(), "Login page visible.");
+        final boolean isHeaderDisplayed = driver.findElement(LOGIN_HEADER).isDisplayed();
+        final boolean isLoginFormDisplayed = driver.findElement(LOGIN_FORM).isDisplayed();
+        Assert.assertTrue(isLoginFormDisplayed, "Validate login form is displayed");
+        Assert.assertTrue(isHeaderDisplayed, "Validate header is displayed");
+    }
+
+    public void enterCredentials(final String email, final String password) {
+        driver.findElement(EMAIL_INPUT).sendKeys(email);
+        driver.findElement(PASSWORD_INPUT).sendKeys(password);
+    }
+
+    public void clearLoginFormInput() {
+        driver.findElement(EMAIL_INPUT).clear();
+        driver.findElement(PASSWORD_INPUT).clear();
+    }
+
+    public void clickLoginSubmitButton() {
+        driver.findElement(LOGIN_SUBMIT_BTN).click();
+    }
+
+    public void errorMessageInvalidCredentials() {
+
+        final boolean isErrorMessageDisplayed = driver.findElement(ACTUAL_ERROR_MESSAGE_INVALID_CRED).isDisplayed();
+        Assert.assertTrue(isErrorMessageDisplayed, "Validate error message is displayed");
+        final String actualErrorMessage = driver.findElement(ACTUAL_ERROR_MESSAGE_INVALID_CRED).getText();
+        final String expectedErrorMessage = "Invalid credentials!";
+        Assert.assertEquals(actualErrorMessage, expectedErrorMessage, "Validate error message is correct");
+    }
+
+    public void clickForgotPasswordField() {
+        driver.findElement(FORGOT_PASSWORD_FIELD).click();
+    }
+
+    public void validateForgotPasswordPageOpened(final String email) {
+
+        Assert.assertTrue(driver.findElement(CHANGE_YOUR_PASSWORD).isDisplayed());
+        Assert.assertTrue(driver.findElement(LET_FIND_YOUR_ACCOUNT).isDisplayed());
+    }
+
+    public void enterEmailForgotPWDandClick(String email) {
+
+        driver.findElement(EMAIL_INPUT).sendKeys(email);
+
+        driver.findElement(FORGOT_PWD_CONTINUE_BUTTON).click();
+    }
```

Comment:
seems like PWD is a whole another page, this method should not be part of the login page